tests/event_threads/Makefile

```diff
@@ -1,5 +1,5 @@
 include ../Makefile.tests_common
 
-USEMODULE += event_thread_highest event_thread_medium event_thread_lowest
+USEMODULE += event_thread_highest event_thread_medium
```

Comment:
Just for my understanding: if only `event_thread` is selected, only a single event thread is created, right?